lib/core/src/display/camera/camera2d.rs

```diff
@@ -115,8 +118,8 @@ struct Camera2dData {
     view_projection_matrix : Matrix4<f32>,
     projection_dirty       : ProjectionDirty,
     transform_dirty        : TransformDirty2,
-    #[derivative(Debug="ignore")]
-    zoom_update_callback   : Option<ZoomUpdateCallback>
+    zoom_update_registry   : XCallbackRegistry1<f32>,
```

Comment:
What does `X` stand for?

Reply:
@mwu-tow extension? They are coexisting with CallRegistry1. I think it's supposed to replace CallbackRegistry. We need to plan a refactoring task to standardize API and do some clean up on duplicated work.